sw/airborne/modules/obstacle_avoidance/obstacle_avoidance.h

```diff
@@ -70,4 +29,21 @@ extern float c5_oa;
 extern float kv;
 extern float epsilon;
 
+extern void serial_init(void);
+extern void serial_update(void);
+extern void serial_start(void);
+void setAnglesMeasurements(float *anglesMeasurements, float *anglesMeasurements_ver, float *centersensorRad, float *fieldOfViewRad,
+                           uint16_t *size_matrix_local);
+extern void matrix_2_pingpong(float *distancesMeters, uint16_t *size_matrix, float *distances_hor);
+extern void kalman_filter_distances(float *distancesMeters);
```

Comment:
How does this function know how many distances there are in the array? Can we add an integer that says how long this array is?